charts/astronomer/templates/commander/commander-deployment.yaml

```diff
@@ -126,6 +126,22 @@ spec:
             - name: COMMANDER_MANAGE_NAMESPACE_RESOURCE
               value: "false"
             {{- end }}
+            - name: COMMANDER_DATAPLANE_CHART_VERSION
+              value: {{ .Values.commander.dataplaneChartVersion | default "" | quote }}
+            - name: COMMANDER_CLOUD_PROVIDER
+              value: {{ .Values.commander.cloudProvider | default "" | quote }}
+            - name: COMMANDER_VERSION
+              value: {{ .Values.images.commander.tag | default "" | quote }}
+            - name: COMMANDER_URL
+              value: {{ .Values.commander.commanderUrl | default "" | quote }}
+            - name: COMMANDER_DATAPLANE_URL
+              value: {{ .Values.commander.dataplaneUrl | default "" | quote }}
+            - name: COMMANDER_DATAPLANE_ID
+              value: {{ .Values.commander.dataplaneId | default (randAlphaNum 8) | quote }}
+            - name: COMMANDER_REGION
+              value: {{ .Values.commander.region | default "" | quote }}
+            - name: COMMANDER_BASE_DOMAIN
+              value: {{ .Values.global.baseDomain | default "" | quote }}
```

Comment:
we need below env vars 
 COMMANDER_DATAPLANE_MODE=True|False , COMMANDER_HOUSTON_AUTHORIZATION_URL=